### DISCUSSION TITLE
ローカルでの起動・バグチェック

### DISCUSSION THREAD

**Discussion by PrefKarafuto** [PrefKarafuto/ex0ch#26] [Announcements]
ローカル環境下でのぜろちゃんねるプラス起動方法(Windows)について説明します。

**Comment by PrefKarafuto** [comment]
必要なもの
AN HTTPD
https://web.archive.org/web/20140314135237/http://www.st.rim.or.jp/~nakata/httpd142p.zip
ActivePerl
http://www.activestate.com/activeperl/

参考サイト
https://kurage.ready.jp/jhp_g/susumu/cgiset.html
こちらを参考にツールのDLとセットアップを進めてもらえればOKです。

**Reply by PrefKarafuto** [reply]
エラーが出た場合は状況を教えていただければ可能な限り協力いたします。

**Reply by Kusaimara** [reply]
XAMPP
https://www.apachefriends.org/jp/index.html

もしくはLaragon+Strawberry Perlでローカル環境を構築すると手軽です。
https://laragon.org/
https://strawberryperl.com/

**Comment by PrefKarafuto** [comment]
参考：当環境で出たエラー
  
Expat.xs.dllが見つからない
------------
```
Can't load 'C:/Strawberry/perl/vendor/lib/auto/XML/Parser/Expat/Expat.xs.dll' for module
```
解決情報:https://stackoverflow.com/questions/34838582/perl-cant-load-expat-xs-dll

ブラウザのcgi起動画面で、モジュールが見つからない云々のエラー表示又は500 Internal Server Error
------------
解決情報：500エラーの場合はadmin.cgiの
```
##use CGI::Carp qw(fatalsToBrowser warningsToBrowser);
```
の##を抜いて上書き保存＝＞足りないと言われてるモジュールをインストール
ppmでインストール出来ない場合はcpanから。

cc1.exeのエラー
---------------
gccの最新版をインストール&(システム環境変数にPathを通す)
